oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java

```diff
@@ -132,11 +139,29 @@ public class MongoDocumentStore implements DocumentStore {
 
     private static final Bson BY_ID_ASC = new BasicDBObject(Document.ID, 1);
 
+    private static final String OPLOG_RS = "oplog.rs";
+
+    /**
+     * The threshold value after which the document store should start (if enabled) throttling.
+     */
+    // For mongo based document store this value is threshold for the oplog replication window.
+    public static final int DEFAULT_THROTTLING_THRESHOLD = Integer.getInteger("oak.mongo.throttlingThreshold", 2);
```

Comment:
Can you please merge the two comments into a single block comment? I assume the unit for this value is hours, but I'm not sure. Can you please clarify with a sentence in the comment?